<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frameHandler</class>
 <widget class="QWidget" name="frameHandler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>169</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="wrapperLayout">
   <item>
    <layout class="QGridLayout" name="frameHandlerLayout" rowstretch="0,0,0,0" columnstretch="0,1,0,1">
     <item row="3" column="2" colspan="2">
      <widget class="QComboBox" name="frameSizeComboBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="widthSpinBox"/>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="heightSpinBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Frame Size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>widthSpinBox</tabstop>
  <tabstop>heightSpinBox</tabstop>
  <tabstop>frameSizeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
